### PR TITLE
IIS Hosting - Server Setup

### PR DESCRIPTION
**Pull request by sofietoft**
Updated .NET references to the latest version.
Also made some minor changes to the structure.

Related to: https://github.com/umbraco/UmbracoDocs/issues/4811

### CODE REVIEW

File: umbraco-cms/fundamentals/setup/server-setup/iis.md [hunk 1]
```diff
@@ -1,14 +1,14 @@
...
 
-# Hosting Umbraco 9+ in IIS
+# Hosting Umbraco in IIS
 
-## Configuring IIS for .NET 5
+## Configuring IIS for .NET
 
-- Install the ["ASP.NET Core Runtime"](https://dotnet.microsoft.com/en-us/download/dotnet/5.0) and download the **Hosting Bundle**.
-- Once you have the hosting bundle installed and have restarted IIS (`net stop was /y` followed by `net start w3svc`), create a site in IIS as you would for a v8 site, however you need to ensure that ".NET CLR version" is set to "No Managed Code" for the Application Pool.
+- Install [ASP.NET Core Runtime](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) and download the **Hosting Bundle**.
```
Comment (eshanrnh) [inline umbraco-cms/fundamentals/setup/server-setup/iis.md:9]:
```suggestion
- Install the [ASP.NET Core Runtime](https://dotnet.microsoft.com/en-us/download/dotnet/7.0) and download the **Hosting Bundle**.
```

File: umbraco-cms/fundamentals/setup/server-setup/iis.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
+- Restart IIS (`net stop was /y` followed by `net start w3svc`)
+- Create a site in IIS-and ensure that the .NET Common Language Runtim (CLR) version` is set to `No Managed Code` for the Application Pool.
```
Comment (eshanrnh) [inline umbraco-cms/fundamentals/setup/server-setup/iis.md:11]:
```suggestion
- Create a site in IIS and ensure that the .NET Common Language Runtime (CLR) version is set to `No Managed Code` for the Application Pool.
```

File: umbraco-cms/fundamentals/setup/server-setup/iis.md [hunk 3]
```diff
@@ -20,9 +20,9 @@ You can use the dotnet CLI to compile and collate all files required for hosting
 dotnet publish -o ../deployment-artefacts -f net5.0
 ```
 
-Alternatively you can use folder or ftp publishing in Visual Studio to compile and collate all required files to for the application to run.
+Alternatively you can use File Transfer Protocal (FTP) publishing in Visual Studio to compile and collate all required files to for the application to run.
```
Comment (eshanrnh) [inline umbraco-cms/fundamentals/setup/server-setup/iis.md:23]:
```suggestion
Alternatively, you can use the File Transfer Protocol (FTP) publishing in Visual Studio to compile and collate all the required files for the application to run.
```

File: umbraco-cms/fundamentals/setup/server-setup/iis.md [hunk 4]
```diff
@@ -20,9 +20,9 @@ You can use the dotnet CLI to compile and collate all files required for hosting
...
 
-In Visual Studio right click on Umbraco web project in the *Solution Explorer* and choose *Publish...* command.
+In Visual Studio right, select the Umbraco web project in the *Solution Explorer* and choose the *Publish...* command.
```
Comment (eshanrnh) [inline umbraco-cms/fundamentals/setup/server-setup/iis.md:25]:
```suggestion
In Visual Studio, select the Umbraco web project in the *Solution Explorer* and choose the *Publish...* command.
```

File: umbraco-cms/fundamentals/setup/server-setup/iis.md [hunk 5]
```diff
@@ -35,27 +35,27 @@ In the *Management* section you find the *Configuration Editor*:
...
 ### IIS Hosting models
 
-IIS can host .NET 5 applications using 2 different hosting models
+IIS can host .NET applications using 2 different hosting models
 
-- [In-process (default)](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/in-process-hosting?view=aspnetcore-5.0)
-- In-process hosting runs an .NET 5 app in the same process as its IIS worker process
-- [Out-of-process](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/out-of-process-hosting?view=aspnetcore-5.0) - to enable this model you need to edit your .csproj file and add:
+- [In-process (default)](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/in-process-hosting?view=aspnetcore-7.0)
+- In-process hosting runs an .NET app in the same process as its IIS worker process
```
Comment (eshanrnh) [inline umbraco-cms/fundamentals/setup/server-setup/iis.md:52]:
```suggestion
- In-process hosting runs a .NET app in the same process as its IIS worker process
```

File: umbraco-cms/fundamentals/setup/server-setup/iis.md [hunk 6]
```diff
@@ -35,27 +35,27 @@ In the *Management* section you find the *Configuration Editor*:
...
+- [Out-of-process](https://docs.microsoft.com/en-us/aspnet/core/host-and-deploy/iis/out-of-process-hosting?view=aspnetcore-7.0) - to enable this model you need to edit your .csproj file and add:
 
 ```js
 <PropertyGroup>
   <AspNetCoreHostingModel>OutOfProcess</AspNetCoreHostingModel>
 </PropertyGroup>
 ```
 
-Out-of-process .NET 5 apps run in a separate from the IIS worker process. The module controls the management of the Kestrel server and requests are proxied between them.
+Out-of-process .NET apps run in a separate from the IIS worker process. The module controls the management of the Kestrel server and requests are proxied between them.
```
Comment (eshanrnh) [inline umbraco-cms/fundamentals/setup/server-setup/iis.md:61]:
```suggestion
Out-of-process .NET apps run separately from the IIS worker process. The module controls the management of the Kestrel server and requests are proxied between them.
```

**Review by eshanrnh — APPROVED** [review]
Looks good, @sofietoft 🙌 I've added some minor comments.

**Review by sofietoft — COMMENTED** [review]
Thanks for the review @eshanrnh 
Added all suggestions 💪

**Pull request merged**